### PR TITLE
:book:  add dex quickstart

### PR DESCRIPTION
**Pull request by mjudeikis**
## Summary

Quick readme on how to run kcp with local dex instance for closer to real-life development. 

## Related issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by mjudeikis** [comment]
/hold

few typos

**Comment by mjudeikis** [comment]
Relates to this:
https://github.com/dexidp/dex/issues/1080

if we want to test locally and be "as close to production" as you can - we need simple group support. For now cherry-picked to my fork, rebased and pointing to fork.

File: contrib/kcp-dex/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+How to run local kcp with dex.
+
+## Step by step guide
+
+### Dex
+
+Run dex outside of kcp
+We use dex to manage OIDC, following the steps below you can run a local OIDC issuer using dex:
+
+* First, clone the dex repo: `git clone https://github.com/mjudeikis/dex.git -b mjudeikis/groups.support`
```
Comment (mjudeikis) [inline contrib/kcp-dex/README.md:12]: this is now using fork so we can do simple groups support.

File: contrib/kcp-dex/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+* Run dex: `./bin/dex serve ../contrib/kcp-dex/kcp-config.yaml `
+
+
+### KCP
+
+Start kcp with oidc enabled:
+
+```bash
+o run ./cmd/kcp start \
```
Comment (embik) [inline contrib/kcp-dex/README.md:31]:
```suggestion
go run ./cmd/kcp start \
```

**Pull request merged**